src/components/menu/_menu.js

```diff
@@ -259,16 +259,16 @@ function MenuController($mdMenu, $attrs, $element, $scope) {
    * the offset of top and left in pixels.
    */
   function offsets() {
-    var offsets = ($attrs.mdOffset || '0 0').split(' ').map(parseFloat);
-    if (offsets.length == 2) {
+    var topLeftOffsets = ($attrs.mdOffset || '0 0').split(' ').map(parseFloat);
```

Comment:
@Splaktar - let's resist these types of changes in the future; as they only add noise to the diff log while offering minimal clarity improvements.